### DISCUSSION TITLE
Welcome to contribute-practice Discussions!

### DISCUSSION THREAD

**Discussion by HyunCafe** [HyunCafe/contribute-practice#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cassiofb-dev** [comment]
Hi, my name is Cássio Fernando!

I'm a Brazilian fullstack dev that loves to talk and share about my passions and I hope to discuss tech stuff here.

Things I love (in order of preference 🤣):

1. Technology
   1. Coding
      1. [TypeScript](https://www.typescriptlang.org/docs/handbook/intro.html)
      2. [JavaScript](https://tc39.es/ecma262/)
      3. [Python](https://docs.python.org/3/)
      4. [C/C++](https://www.gnu.org/software/libc/manual/html_mono/libc.html)
      5. [Java](https://docs.oracle.com/en/java/)
   2. [Linux](https://github.com/torvalds/linux)
      1. [Void Linux](https://voidlinux.org/)
      2. [OpenSUSE Thumbleweed](https://get.opensuse.org/tumbleweed/)
      3. [Manjaro](https://manjaro.org/)
      4. [ZorinOS](https://zorin.com/os/)
      5. [Ubuntu](https://ubuntu.com/)
   3. Hardware
      1. AMD
      2. Intel
      3. Qualcomm
      4. Mediatek
      5. [NVIDIA](https://www.youtube.com/watch?v=i2lhwb_OckQ)
2. Anime
   1. [Tengen Toppa Gurren Lagann](https://kitsu.io/anime/tengen-toppa-gurren-lagann)
   2. [One Piece](https://kitsu.io/anime/one-piece)
   3. [Clannad](https://kitsu.io/anime/clannad)
   4. [Hunter X Hunter](https://kitsu.io/anime/hunter-x-hunter)
   5. [Ranking of Kings](https://kitsu.io/anime/ousama-ranking)
3. Anime Openings
   1. [Ranking of Kings: Opening 2](https://www.youtube.com/watch?v=sLZ-BfWaWXc)
   2. [Shigatsu Wa Kimi no Uso: Opening 1](https://www.youtube.com/watch?v=sXb3nBEMyMU)
   3. [One Piece: Opening 1](https://www.youtube.com/watch?v=6zH9J3o-4CU)
   4. [One Piece: Opening 22](https://www.youtube.com/watch?v=2moA4y-oVUg)
   5. [Ya Boy Kongming!: Opening 1](https://www.youtube.com/watch?v=gNn9NxZH2Vo)
4. Games
   1. [Pokémon](https://en.wikipedia.org/wiki/Pok%C3%A9mon_(video_game_series))
   2. [Genshin Impact](https://genshin.hoyoverse.com/en)
   3. [Terraria](https://www.terraria.org/)
   4. [Minecraft](https://www.minecraft.net/en-us)
   5. [Chess](https://lichess.org/)
5. Music
   1. Eletronic
      1. [Sphere](https://www.youtube.com/watch?v=0ZUoFPLlVQU)
      2. [Reanimate](https://www.youtube.com/watch?v=OmWkLOku8pQ)
      3. [Scaramouche Boss Theme](https://www.youtube.com/watch?v=8HwSW2AkLo0)
      4. [Yatagarasu](https://www.youtube.com/watch?v=ag1ayovdaxc)
      5. [Deadlocked](https://www.youtube.com/watch?v=OPBECnDBiRQ)
   2. Instrumental
      1. [Moonlight Sonata](https://www.youtube.com/watch?v=4591dCHe_sE)
      2. [Four Seasons](https://www.youtube.com/watch?v=GRxofEmo3HA)
      3. [Waldstein Sonata](https://www.youtube.com/watch?v=GRxofEmo3HA)
      4. [Rondo Alla Turca](https://www.youtube.com/watch?v=aeEmGvm7kDk)
      5. [Nocturne](https://www.youtube.com/watch?v=p29JUpsOSTE)
   3. Rock
      1. [Sultans of Swing](https://www.youtube.com/watch?v=8Pa9x9fZBtY)
      2. [Bohemian Rhapsody](https://www.youtube.com/watch?v=fJ9rUzIMcZQ)
      3. [Made in Heaven](https://www.youtube.com/watch?v=171skzi5BKc)
      4. [Stairway to Heaven](https://www.youtube.com/watch?v=X791IzOwt3Q)
      5. [One](https://www.youtube.com/watch?v=WM8bTdBs-cw)
   4. Japanese
      1. [Hakada no Yuusha](https://www.youtube.com/watch?v=FT0GKCuSaW0)
      2. [Hikaru Nara](https://www.youtube.com/watch?v=IeJTNN8_jLI)
      3. [Welcome](https://www.youtube.com/watch?v=BK0tDxrzUOc)
      4. [Black Catcher](https://www.youtube.com/watch?v=hP6VM6YAMIE)
      5. [Black Rover](https://www.youtube.com/watch?v=VSucxMB_Z6k)
   5. Pop
      1. [The Scientist](https://www.youtube.com/watch?v=RB-RcX5DS5A)
      2. [Photograph](https://www.youtube.com/watch?v=nSDgHBxUbVQ)
      3. [RISE](https://www.youtube.com/watch?v=fB8TyLTD7EE)
      4. [Enemy](https://www.youtube.com/watch?v=D9G1VOjN_84)
      5. [See Tinh](https://www.youtube.com/watch?v=tVTRY6851Ug)
